### ISSUE TITLE
apk Builder

### ISSUE THREAD

**Issue by micfio** [tranleduy2000/pascalnide#21]
Is possible with the ide create a standalone apk of the app created ?
Thanks

**Comment by VAR-RAV** [comment]
Its a good questions...  Need .apk  create on Android  Devices !  ???
  And what a file made on this IDE (*.DAT, *.DLL, *.???) ? Or only *.pas ?????

**Comment by upiter20** [comment]
Yes. It is a very good questions... Need to create .apk app under Android Devices too!
